SECURITY.MD

```diff
@@ -0,0 +1,41 @@
+<!-- BEGIN MICROSOFT SECURITY.MD V0.0.2 BLOCK -->
```

Comment:
+1, don't know if we need this

Reply:
I almost removed it, but it tells us the version, and (afaik) it won't show up